Belot/Belot.Server/Data/ApplicationDbContext.cs

```diff
@@ -20,7 +22,10 @@ public ApplicationDbContext(DbContextOptions options)
 
         protected override void OnModelCreating(ModelBuilder builder)
         {
-            builder.Entity<ApplicationUser>()
+            base.OnModelCreating(builder);
+
+            builder
```

Comment:
Better define the relationships in new IEntityTypeConfiguration class for consistency (refer to Data/Configurations).

Reply:
```
builder
    .Entity<ApplicationUser>()
    .HasOne(x => x.UserBalance)
    .WithOne(x => x.User)
    .HasForeignKey<UserBalance>(x => x.UserId)
    .IsRequired(false);
```
This is not added into ApplicationUserConfiguration, ApplicationUserConfiguration class was created by me. We can add this existing configuration to ApplicationUserConfiguration ? What do you think ?